src/App.vue

```diff
@@ -380,18 +421,20 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // reset errors in case this method is invoked more than once (ie, retry)
     this.resetFlags()
 
+    // Feature flag safety check
+    let supportedFilings = await getFeatureFlag('supported-filings')
+    if (!supportedFilings?.includes(this.$route.meta.filingType)) {
+      this.accountAuthorizationDialog = true
+    }
+
+    // Protect the routes not associated with this filing type.
+    if (this.$route.meta.filingType !== this.getFilingType) {
+      this.accountAuthorizationDialog = true
```

Comment:
Again, not an account auth problem.

Maybe you could store a "reason text" so that you could reuse this dialog? Or create a new dialog.

Reply:
Updated with new dialog in next commit.